web/pandas_web.py

```diff
@@ -317,7 +317,20 @@ def roadmap_pdeps(context):
         ) as f:
             json.dump(pdeps, f)
 
-        for pdep in sorted(pdeps["items"], key=operator.itemgetter("title")):
+        regex = r"^PDEP-(\d+)"
+        compiled_pattern = re.compile(regex)
```

Comment:
I'd probably make those two lines into one. Just a personal preference, up to you, but seems unnecessarily verbose.

Reply:
Happy to change. Will ping once done. Cheers.

Reply:
@datapythonista done